configs/Lightning-AI/pl-lite_torch-develop.yaml

```diff
@@ -0,0 +1,35 @@
+target_repository:
+  HTTPS: https://github.com/Lightning-AI/lightning.git
+  # checkout a particular branch or a tag
+  checkout: master
+  # define installing package extras
+  install_extras: dev
+  # copy some  test from the target repository
+  copy_tests:
+    - tests/tests_lite
```

Comment:
These have just been renamed to fabric

Reply:
let's rename it back when it lands on master or we can enable this testing from custom branch :)

Reply:
It has landed already